sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -202,6 +202,7 @@ case class DropTableCommand(
       sparkSession.sharedState.cacheManager.uncacheQuery(
         sparkSession.table(tableName.quotedString))
     } catch {
+      case ae: AnalysisException if ae.getMessage.contains("Table or view not found") =>
```

Comment:
Thank you for review, @gatorsmile .
It's originally `NoSuchTableException`, but `Analyzer` changed into this at [here](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala#L516-L517).
So, the catch expression becomes ugly.